<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Matrix Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 225);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>565</width>
      <height>507</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="TypeBox">
         <property name="font">
          <font>
           <family>Century Gothic</family>
          </font>
         </property>
         <property name="title">
          <string/>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="radioButtonSxM">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Scalar w/ Matrix</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonMxM">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Matrix w/ Matrix</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QGroupBox" name="operationBox">
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="title">
              <string>Operation</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QRadioButton" name="radioButtonAdd">
                <property name="font">
                 <font>
                  <family>MS Shell Dlg 2</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Add</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButtonSubtract">
                <property name="font">
                 <font>
                  <family>MS Shell Dlg 2</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Subtract</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButtonMultiply">
                <property name="font">
                 <font>
                  <family>MS Shell Dlg 2</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Multiply</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="font">
                <font>
                 <family>Lucida Console</family>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Scalar:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditSxMScalar">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(253, 254, 238);</string>
               </property>
               <property name="frame">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <family>Lucida Console</family>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Matrix:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Number of Rows</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditSxMRows">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(253, 254, 238);</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Number of Columns</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditSxMCols">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(253, 254, 238);</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Values:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditSxMValues">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(253, 254, 238);</string>
               </property>
               <property name="placeholderText">
                <string>enter csv or autofill</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonAutofillSxM">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="text">
                <string>(Autofill)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonSubmitSxM">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Submit</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <family>Lucida Console</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Matrix A:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Number of Rows</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditMxMARows">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(253, 254, 238);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Number of Columns</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditMxMACols">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(253, 254, 238);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Values:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditMxMAValues">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(253, 254, 238);</string>
             </property>
             <property name="placeholderText">
              <string>enter csv or autofill</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonAutofillMxMA">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>(Autofill)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="font">
              <font>
               <family>Lucida Console</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Matrix B:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Number of Rows</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditMxMBRows">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(253, 254, 238);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Number of Columns</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditMxMBCols">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(253, 254, 238);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Values:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditMxMBValues">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(253, 254, 238);</string>
             </property>
             <property name="placeholderText">
              <string>enter csv or autofill</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonAutofillMxMB">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>(Autofill)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonSubmitMxM">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Submit</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textEditTextOut">
       <property name="font">
        <font>
         <family>Monospac821 BT</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(253, 254, 238);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="midLineWidth">
        <number>1</number>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Monospac821 BT'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="placeholderText">
        <string>Output:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>radioButtonMxM</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditMxMARows</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonMxM</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditMxMACols</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonMxM</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditMxMBRows</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonMxM</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditMxMBCols</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonMxM</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditSxMScalar</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonMxM</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditSxMRows</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonMxM</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditSxMCols</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>flipSxMScalar(Qstring)</slot>
 </slots>
</ui>
